.github/workflows/main.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: "1.17"
+          go-version: "1.18"
```

Comment:
IIRC I think setup-go now has symbolic names for the current and current-1 go releases. Once we catch up with go we can maybe switch to the symbolic name.

Reply:
i just looked. It's  called "oldstable"  (and  "stable" for latest stable)